030-Concepts/080-pricing.mdx

```diff
@@ -7,26 +7,228 @@ slug: concepts/pricing
+- Malformed updates: Rollback changes from updates that led to unexpected or undesired results.
+- Security breaches: In the event of data compromise or unauthorized alterations, revert the database to a secure, pre-breach state.
+
+PITR operates by consistently backing up transaction logs. It grants you the capability to revert data to any designated point within the specified retention window.
+
+### Rate limit
+
+Each shared Xata instance has a set request rate limit to maintain consistent performance for all users. If requests surpass this threshold, they will return a 429 HTTP status. The following limits apply:
+
+- **Free tier:** Limited to 75 requests/second (qps).
+- **Pro tier:** Starts with a 150 requests/second (qps), which can be adjusted upon request.
+
+The Pro plan has a flexible limit, while the Free plan has a strict, fixed limit.
+
+These limits are designated for each branch in the workspace.
+
+## Pricing plans
+
+### Common features across all plans
+
+All Xata plans, irrespective of their pricing, include the following features:
+
+- **PostgreSQL database:** Xata uses Postgres, which serves as the backbone for storing structured, relational data securely. Each plan starts with 10 branches and 15 GB of storage space, but you can get more space if you need it.
+
+- **Unlimited records:** For all our plans, there are no restrictions on the number of records. You can store as many records as your allocated GB permits.
+
+- **High availability (HA):** Consistent service access with minimal interruptions or downtime.
+
+- **Elasticsearch integration:** Integrates efficient and responsive full-text search capabilities within your projects. Our search engine uses Elasticsearch, which makes searches easy because it replicates all the data from our database. You get 15 GB of space initially, but can add more depending on what you need.
+
+- **File attachments:** Xata uses S3 and Cloudflare's CDN for storing and delivering files. Your files are stored safely and you can access them quickly when you need them. Starting storage is 2 GB, but you can add more or customize it per your needs.
+
+- **Image transformations:** You can do 10k image transformations from the start and add more as needed.
+
+- **AI questions:** Xata offers an option for integrating with OpenAI and ChatGPT, allowing you to ask up to 250 AI-powered questions. If you want to ask more questions, you can add more with the Pro or Enterprise plans.
+
+- **Support:** Depending on your plan, you get different levels of support, starting from community support to dedicated support.
+
+### Free plan
+
+The Free plan is perfect for individuals who are working on personal projects or are in the experimentation phase. This plan offers the basic functionalities needed to get started, at no cost.
+
+**The Free plan is ideal for:**
+
+- Individual developers
+- Students
+- Hobbyists
+- Experimenters who want to learn more about Xata
+
+##### Features
```

Comment:
At the moment, we don't offer many differentiating features between the plans. This will change, and eventually will have more value add between the tiers, but it's primarily usage based right now. I'd either opt out of features or focus on the usage / consumption pricing for the pro tier instead.